src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,629 @@
+  'Peru',
+  'Philippines',
+  'Philippines (restricted payments)',
+  'Poland',
+  'Portugal',
+  'Qatar',
+  'Republic of Yemen',
+  'Romania',
+  'Russia',
+  'Rwanda',
+  'Sao-Tome/Principe',
+  'Saudi Arabia',
+  'Scotland',
+  'Senegal',
+  'Serbia',
+  'Serbia/Montenegro',
+  'Seychelles',
+  'Sicily',
+  'Sierra Leone',
+  'Singapore',
+  'Slovakia',
+  'Slovenia',
+  'Somalia',
+  'South Africa',
+  'South Korea',
+  'Spain',
+  'Sri Lanka',
+  'St. Kitts',
+  'St. Lucia',
+  'St. Vincent',
+  'Sudan',
+  'Suriname',
+  'Swaziland',
+  'Sweden',
+  'Switzerland',
+  'Syria',
+  'Taiwan',
+  'Tajikistan',
+  'Tanzania',
+  'Thailand',
+  'Togo',
+  'Trinidad and Tobago',
+  'Tunisia',
+  'Turkey (Adana only)',
+  'Turkey (except Adana)',
+  'Turkmenistan',
+  'Uganda',
+  'Ukraine',
+  'United Arab Emirates',
+  'United Kingdom',
+  'Uruguay',
+  'Uzbekistan',
+  'Vanuatu',
+  'Venezuela',
+  'Vietnam',
+  'Wales',
+  'Western Samoa',
+  'Yemen Arab Republic',
+  'Zambia',
+  'Zimbabwe'
+];
+
+export const militaryPostOfficeTypeLabels = { // TODO: determine whether these are necessary
+  APO: 'Army Post Office',
+  FPO: 'Fleet Post Office',
+  DPO: 'Diplomatic Post Office'
+};
+
+const addressUISchema = (addressName, title) => {
+  return {
+    'ui:title': title,
+    type: {
+      'ui:title': 'Type',
+      'ui:options': {
+        labels: typeLabels
+      }
+    },
+    country: {
+      'ui:title': 'Country',
+      'ui:options': {
+        labels: {
+          USA: 'United States'
+        }
+      }
+    },
+    state: {
+      'ui:title': 'State',
+      'ui:options': {
+        labels: stateLabels,
+        hideIf: formData => {
+          return (
+            _.get(formData, `veteran[${addressName}].country`) !== 'USA'
+          );
+        }
+      }
+    },
+    addressLine1: {
+      'ui:title': 'Street'
+    },
+    addressLine2: {
+      'ui:title': 'Line 2'
+    },
+    addressLine3: {
+      'ui:title': 'Line 3'
+    },
+    city: {
+      'ui:title': 'City'
+    },
+    militaryStateCode: {
+      'ui:title': 'Military State Code',
+      'ui:options': {
+        labels: stateLabels,
+        hideIf: formData => _.get(formData, `veteran[${addressName}].type`) !== 'MILITARY'
+      }
+    },
+    zipCode: {
+      'ui:title': 'ZIP code',
+      'ui:validations': [validateZIP],
+      'ui:errorMessages': {
+        pattern: 'Please enter a valid 5 or 9 digit ZIP code (dashes allowed)'
+      },
+      'ui:options': {
+        widgetClassNames: 'va-input-medium-large',
+        hideIf: formData =>
+          _.get(formData, `veteran[${addressName}].type`) !== 'DOMESTIC'
+      }
+    },
+    militaryPostOfficeTypeCode: {
+      'ui:title': 'Military Post Office Type Code',
+      'ui:options': {
+        labels: militaryPostOfficeTypeLabels,
+        hideIf: formData => _.get(formData, `veteran[${addressName}].type`) !== 'MILITARY'
+      }
+    }
+  };
+};
+
+const addressSchema = (isRequired = false) => {
```

Comment:
Yep, all of the address logic will be moved into the platform address file, so it can be located with the helpers it will use, and be shared more widely once it's set up.